src/gui/views/FolderCell.java

```diff
@@ -16,9 +20,16 @@ public void updateItem(final Folder folder, boolean empty) {
 
         if (folder == null) { return; }
 
-        Button button = new Button(folder.getName());
+        final ImageView imageView = new ImageView(
+        	      new Image("http://www.iconsdb.com/icons/preview/gray/folder-xxl.png")
+        	    );
+        imageView.setFitHeight(40);
+        imageView.setFitWidth(40);
+        Button button = new Button(folder.getName(),imageView);
         // Add class name to this button to apply styles from the CSS file.
         button.getStyleClass().add("folder");
+        button.setAlignment(Pos.BASELINE_LEFT);
+        button.setId("folder");
```

Comment:
I think this line and line 29 are the same, here you are setting an ID and in line 29 we are setting a class name. Delete one of them. (I think you may want to delete line 29).